packages/react-jsx-highcharts/src/components/Tooltip/Tooltip.js

```diff
@@ -13,7 +13,7 @@ class Tooltip extends Component {
   };
 
   static defaultProps = {
-    ...Highcharts.defaultOptions.tooltip,
+    ...(Highcharts.defaultOptions && Highcharts.defaultOptions.tooltip),
```

Comment:
Would you mind applying the same fix to [RangeSelector](https://github.com/whawker/react-jsx-highcharts/blob/master/packages/react-jsx-highstock/src/components/RangeSelector/RangeSelector.js#L16)?